packages/backend/src/errors/unauthorisedException.ts

```diff
@@ -0,0 +1,7 @@
+import { BaseError } from './baseError'
+export class UnauthorisedException extends BaseError {
```

Comment:
I think it will be better to have errors/exceptions in each module, and here only some generic once
also if the file is a class I'll propose to have the class name as file name (starting with capital)

Reply:
Hmm, I agree. I only kept it all in a single common error directory because there were only 2 extensions of the BaseError for now.

I will move the Unauthorised exception in the auth module.
BadRequest should be for every module, and as such I will keep it here along with the BaseError